src/Microsoft.Health.Operations/OperationState.cs

```diff
@@ -58,8 +58,14 @@ public class OperationState<T>
     public IReadOnlyCollection<Uri>? Resources { get; init; }
 
     /// <summary>
-    /// Gets the optional collection of operation-specific properties.
+    /// Gets the optional results of the operation.
     /// </summary>
-    /// <value>Zero or more key-value pairs based on the <see cref="Type"/>.</value>
-    public IReadOnlyDictionary<string, string>? AdditionalProperties { get; init; }
+    /// <remarks>
+    /// The results may change over time as the operation continues execution.
+    /// </remarks>
+    /// <value>
+    /// An object whose type depends on the <see cref="OperationState{T}.Type"/> if specified;
+    /// otherwise <see langword="null"/>.
+    /// </value>
+    public object? Results { get; init; }
```

Comment:
How do people like `Results`? Other options could be `Result`, `Output`, `Metadata`, or something else?

E.g.
```json
{
    "CreatedTime": "8/7/2022 01:23:45Z",
    "Status": "Running",
    "PercentComplete": 42,
    "Results": {
        "Exported": 1000,
        "Failed": 2,
        "ErrorHref": "foo"
    }
}
```

Reply:
For export, I was thinking of renaming `"Failed"` to `"Skipped"`, but always keeping it present in the response even if it's `0`.

When you say `OperationStatus`, are you referring to the enumeration? Could you give an example of what you mean? For clarity's sake, the above example would be a sample response to a use that queried the state of their operation via `GET /operations/{operation-id}`

Reply:
I think our hands are a little tied on some of the structure according to the [REST API Guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#percent-complete). Their structure appears to be flat for the generic operation properties
```json
{
  "createdDateTime": "2015-06-19T12-01-03.45Z",
  "percentComplete": "50",
  "status": "running"
}
```

However, I think our options become whether to include operation-specific metadata flat at the top-level or nest it under some top-level property, as the specification does not explicitly talk about it. I am leaning towards encapsulating the operation-specific metadata in a single property to neatly organize it.

Reply:
Although maybe in the most RESTful world we would have no operation-specific metadata and instead that metadata would be made manifest through another resource. Like you could `GET` an export from like `/export/<id>` to retrieve metadata

Reply:
Based on our discussions, an additional resource seems unnecessary at this time